### DISCUSSION TITLE
Does ASK have inline autofill?

### DISCUSSION THREAD

**Discussion by RobioEpo** [AnySoftKeyboard/AnySoftKeyboard#3039] [Q&A] [answered]
Does AnySoftKeyboard have inline autofill for passwords?

**Comment by menny** [comment, marked as answer]
It does. In the beta channel at the moment

**Reply by RobioEpo** [reply]
Thats cool to know its in the works at the moment. I'll see if I can download the beta version and test it out. 

Thanks @menny

**Reply by RobioEpo** [reply]
> It does. In the beta channel at the moment

Any time frame for when it will release to stable version?

**Reply by menny** [reply]
It is rolling out right now to Play Store
